### PR TITLE
update bnbchain repo

### PR DESCRIPTION
**Pull request by CocoStarZ**

### CODE REVIEW

File: data/ecosystems/b/bnb-chain.toml [hunk 1]
```diff
@@ -470,9 +470,58 @@ github_organizations = [
...
+
+[[repo]]
+url = "https://github.com/secondlife"
+
+[[repo]]
+url = "https://github.com/okx"
+
+[[repo]]
+url = "https://github.com/Particle-Network"
+
```
Comment (jubos) [inline data/ecosystems/b/bnb-chain.toml:506]: These are either users or organizations and not repos.  You should either make sub ecosystems for them or hook up the existing sub ecosystems to BNBChain if they should be attributed to BNB.

**Review by jubos — CHANGES_REQUESTED** [review]

**Comment by CocoStarZ** [comment]
Reviews resolved.

File: data/ecosystems/b/bnb-chain.toml [hunk 2]
```diff
@@ -470,9 +470,42 @@ github_organizations = [
...
+  "https://github.com/Space-ID/",
+  "https://github.com/ApeSwapFinance",
+  "https://github.com/carv-protocol",
+  "https://github.com/DEVHooked",
+  "https://github.com/StarryNift",
+  "https://github.com/myshell-ai",
+  "https://github.com/galxe",
+  "https://github.com/secondlife",
+  "https://github.com/okx",
+  "https://github.com/Particle-Network",
```
Comment (jubos) [inline data/ecosystems/b/bnb-chain.toml:487]: Hi, please break these out into sub ecosystems files if they don't exist or use the existing names like "BakeryToken".  A lot of these github orgs roll up into other parent ecosystems, so it keeps the taxonomy more organized.

**Review by jubos — CHANGES_REQUESTED** [review]

**Comment by Hosseinom** [comment]
Add new StarkNet and Cairo related projects

**Pull request closed without merging**